driver-service/src/main/java/com/modsen/driverservice/controllers/AutoController.java

```diff
@@ -0,0 +1,55 @@
+package com.modsen.driverservice.controllers;
+
+import com.modsen.driverservice.dto.AutoDto;
+import com.modsen.driverservice.dto.AutoPageResponse;
+import com.modsen.driverservice.dto.AutoResponseList;
+import com.modsen.driverservice.services.AutoServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/v1/autos")
+@RequiredArgsConstructor
+public class AutoController {
+
+
+    private final AutoServiceImpl autoService;
+
+
+    @GetMapping
+    public ResponseEntity<AutoResponseList> getAll()
+    {
+        return ResponseEntity.ok(autoService.getAll());
+    }
+
+    @GetMapping("/by-number")
+    public ResponseEntity<AutoDto> getByNumber(@RequestParam(name = "number") String number)
+    {
+        return ResponseEntity.ok(autoService.getByNumber(number));
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<AutoDto> getById(@PathVariable(name = "id") Long id)
```

Comment:
Тоже самое, если название переменной и path variable совпадают, то можно явно не указывать в аннотации PathVariable:
```suggestion
    public ResponseEntity<AutoDto> getById(@PathVariable Long id)
```